src/azul/health.py

```diff
@@ -231,16 +231,16 @@ def cached_health(self) -> JSON:
                 body = cache['health']
         return body
 
-    fast_properties: Mapping[str, Set[health_property]] = {
-        'indexer': {
+    fast_properties: Mapping[str, Tuple[health_property]] = {
+        'indexer': (
```

Comment:
Since the reasoning for the changing the properties to tuples isn't obvious, you could consider making it a non-fixup commit that occurs before the rest of the changes.

Giving the new commit a title like "Make ordering of health properties in OpenAPI spec deterministic" would make the reason behind these changes clear.

I think this is optional though.